src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleCloudStorageDAO.scala

```diff
@@ -11,6 +9,8 @@ trait GoogleCloudStorageDAO {
 
   def createBucket(userId: String, projectId: String, bucketName: String): Unit
 
+  // TODO: acls should be BucketAccessControls
```

Comment:
This seems like a tech debt story, either make a story for it or do it, but in either case, remove the todo